font-types/src/fixed.rs

```diff
@@ -1,6 +1,6 @@
 //! fixed-point numerical types
 
-use std::ops::{Add, AddAssign, Sub, SubAssign};
+use core::ops::{Add, AddAssign, Div, DivAssign, Mul, MulAssign, Neg, Sub, SubAssign};
```

Comment:
I have `core` aliased as `std` if we're `no_std`. I'm not sure where I got this pattern from, but I don't think I invented it. In any case, I think we use `std` here and if we want to change the overall pattern we can do that in a separate PR?

Reply:
oh! I actually thought I messed this up by doing automatic imports so this was my attempt to fix it. I have no objections to that pattern so I'll revert this change.